weave

```diff
@@ -810,6 +823,50 @@ dns_args() {
     fi
 }
 
+default_etc_hosts() {
+    NAME=$1
+    shift
+    echo "# created by Weave - BEGIN"
+    echo "# container hostname"
+    for CIDR in $ALL_CIDRS ; do
+        echo "${CIDR%/*}    $NAME"
+    done
+    echo
+    echo "# static names added with --add-host"
+    for EXTRA_HOST in $@ ; do
```

Comment:
I think it is not strictly necessary, as `$@` will be something like `static.name:1.1.1.1 another.name:2.2.2.2`... but I've surrounded it with `"` anyway